client/src/actions/index.js

```diff
@@ -23,52 +23,46 @@ function obsAnnotationFetchAndLoad(dispatch, schema) {
 
   const plimit = new PromiseLimit(5);
   return Promise.all(
-    columns.map((col) =>
-      plimit.add(() => {
-        const path = `annotations/obs?annotation-name=${encodeURIComponent(
-          col.name
-        )}`;
-        const url = `${globals.API.prefix}${globals.API.version}${path}`;
-        return doBinaryRequest(url).then((buffer) => {
-          const df = Universe.matrixFBSToDataframe(buffer);
-          dispatch({
-            type: "universe: column load success",
-            dim: "obsAnnotations",
-            dataframe: df,
-          });
-        });
-      })
-    )
+    columns
+      .filter(col => col.name !== index)
+      .map(col =>
+        plimit.add(() => {
+          const path = `annotations/obs?annotation-name=${encodeURIComponent(col.name)}`;
+          return fetchBinary(path)
```

Comment:
I would standardize on a single style - either embed the path inside the call to `fetchBinary()` and remove the block, or not.  See line 81 and 98 for the opposite style